ironic_oneview_cli/create_port_shell/commands.py

```diff
@@ -33,11 +33,45 @@ def create_port(self, args, ironic_node):
         else:
             mac = self.get_server_hardware_mac(server_hardware)
 
+        if not self.validate_server_hardware_mac(mac, server_hardware):
+            print(("WARNING: mac %(mac)s doesn't match any server "
+                   "hardware's %(sh)s ports") %
+                  {"mac": mac, "sh": server_hardware.get('uuid')})
```

Comment:
Can you add on a new line right after this warning: "Use ironic-oneview port-create command with a valid MAC address to create an Ironic Port for this Ironic Node."